services/continuous_querier/service.go

```diff
@@ -353,6 +355,7 @@ func (s *Service) runContinuousQueryAndWriteResult(cq *ContinuousQuery) error {
 	// Execute the SELECT.
 	ch := s.QueryExecutor.ExecuteQuery(q, influxql.ExecutionOptions{
 		Database: cq.Database,
+		Quiet:    !s.logQueries,
```

Comment:
In general not lining up variable names and then using inverse logic leads to bugs.  It might be better to have change `Quiet` to `LogQuery` and then everything will read nicely and line up for boolean logic.